uPortal-io/uPortal-io-types/src/main/java/org/apereo/portal/io/xml/permission/PermissionSetsDataFunction.java

```diff
@@ -44,6 +44,9 @@
                     + "WHERE NOT (UPP.ACTIVITY = 'SUBSCRIBE' AND UPP.TARGET LIKE '"
                     + IPermission.PORTLET_PREFIX
                     + "%')\n "
+                    + "AND NOT (UPP.ACTIVITY = 'PORTLET_MODE_CONFIG' AND UPP.TARGET LIKE '"
+                    + IPermission.PORTLET_PREFIX
+                    + "%')\n "
```

Comment:
It looks like you added yours correctly;  but this is an opportunity to improve the whole thing, like...

```
private static final String QUERY =
        "SELECT DISTINCT UPP.OWNER, UPET.ENTITY_TYPE_NAME, UPP.PRINCIPAL_KEY, UPP.ACTIVITY, UPP.PRINCIPAL_TYPE\n "
                + "FROM UP_PERMISSION UPP\n "
                + "LEFT JOIN UP_ENTITY_TYPE UPET ON UPP.PRINCIPAL_TYPE = UPET.ENTITY_TYPE_ID\n "
                + "WHERE NOT (UPP.ACTIVITY = '" + IPermssion.PORTLET_SUBSCRIBER_ACTIVITY + "' AND UPP.TARGET LIKE '"
                + IPermission.PORTLET_PREFIX
                + "%')\n "
                + "AND NOT (UPP.ACTIVITY = '" + IPermission.PORTLET_MODE_CONFIG + "' AND UPP.TARGET LIKE '"
                + IPermission.PORTLET_PREFIX
                + "%')\n "
                + "AND NOT (UPP.ACTIVITY = '" + IPermission.PORTLET_BROWSE_ACTIVITY + "' AND UPP.TARGET LIKE '"
                + IPermission.PORTLET_PREFIX
                + "%')\n ";
```